File: core/src/com/oni/donjon/action/Action.java
package com.oni.donjon.action;

import com.oni.donjon.data.GameData;
import com.oni.donjon.map.Tile;
import com.oni.donjon.stage.UIStage;

/**
 * @author Daniel Chesters (on 02/06/14).
 */
public interface Action {
    void doAction(Tile tile, UIStage stage);
}

File: core/src/main/kotlin/com/oni/donjon/action/Action.kt
package com.oni.donjon.action

import com.badlogic.gdx.utils.I18NBundle
import com.oni.donjon.map.Tile
import com.oni.donjon.stage.UIStage

/**
 * @author Daniel Chesters (on 02/06/14).
 */
interface Action {
    fun doAction(tile: Tile, stage: UIStage, bundle: I18NBundle)
}
